.eslintrc.json

```diff
@@ -14,18 +14,10 @@
   },
   {
     "files": [
-      "lang/*.js"
+      "**/lang/*.js"
     ],
     "rules": {
       "quotes": 0
     }
-  },
-  {
-    "files": "mixins/demo/**.html",
-    "rules": {
-      "quotes": 0,
-      "no-undef": 0,
-      "indent": 0
-    }
```

Comment:
This was added [here](https://github.com/BrightspaceUI/core/commit/12fbea8546ea840e54ae0ced2d6cf72955206504), unnecessarily because we reverted the related demo changes.